### ISSUE TITLE
Module not found: Can't resolve 'React' in '/srv/node_modules/semantic-ui-react/dist/umd'

### ISSUE THREAD

**Issue by alycda** [Semantic-Org/Semantic-UI-React#3597]
### Steps
Upgrade to 0.87 from 0.86

1. `yarn upgrade semantic-ui-react@0.87`
2. restart webpack

### Expected Result
webpack dev server compiles

### Actual Result
```
Failed to compile
./node_modules/semantic-ui-react/dist/umd/semantic-ui-react.min.js
Module not found: Can't resolve 'React' in '/srv/node_modules/semantic-ui-react/dist/umd'
This error occurred during the build time and cannot be dismissed.
```

### Version
0.87.0

**EDIT** our webpack config DOES NOT include `{ resolve: { aliasFields: ['browser'] } }`

**Comment by alycda** [comment]
Started with a fresh CRA app, and got: 

```
Failed to compile
./node_modules/semantic-ui-react/dist/umd/semantic-ui-react.min.js
Module not found: Can't resolve 'ReactDOM' in '/Users/max/Desktop/cra-sui/node_modules/semantic-ui-react/dist/umd'
```

### steps to reproduce:
1. `npx create-react-app ./`
2. `yarn add semantic-ui-react`
3. add `import { Button } from 'semantic-ui-react';` to src/App.js

package.json
```
{
  "name": "cra-sui",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "react": "^16.8.6",
    "react-dom": "^16.8.6",
    "react-scripts": "3.0.0",
    "semantic-ui-react": "^0.87.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": "react-app"
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
```

[yarn.lock.zip](https://github.com/Semantic-Org/Semantic-UI-React/files/3149833/yarn.lock.zip)

**Comment by layershifter** [comment]
Confirmed, caused by #3566. `umd` build should not be used by default.

**Comment by layershifter** [comment]
Fixed in #3598, will keep opened until next release.

**Comment by levithomason** [comment]
Release on the way, hang tight...

**Comment by levithomason** [comment]
Fixed in `semantic-ui-react@0.87.1`
